### PR TITLE
Add back BepInEx project sources into LethalAPI.Core.csproj

### PR DESCRIPTION
**Pull request by dhkatz**
Seems like for fresh clones of the repo, the BepInEx dependencies will not always resolve properly without the project sources added by the BepInEx template.

### CODE REVIEW

File: LethalAPI.Core/LethalAPI.Core.csproj [hunk 1]
```diff
@@ -9,6 +9,11 @@
         <DebugType>embedded</DebugType>
         <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
         <LangVersion>latest</LangVersion>
+        <RestoreAdditionalProjectSources>
+            https://api.nuget.org/v3/index.json;
+            https://nuget.bepinex.dev/v3/index.json;
+            https://nuget.samboy.dev/v3/index.json
```
Comment (MaxWasUnavailable) [inline LethalAPI.Core/LethalAPI.Core.csproj:15]: This source doesn't appear to be necessary?

**Review by MaxWasUnavailable — CHANGES_REQUESTED** [review]
Unless that one is a hard requirement, I'd rather we remove any unnecessary sources.

File: LethalAPI.Core/LethalAPI.Core.csproj (hunk 1 above)
Comment (dhkatz) [reply LethalAPI.Core/LethalAPI.Core.csproj]: Removed 👍

**Review by MaxWasUnavailable — APPROVED** [review]
Looks good

**Pull request merged**